driver.go

```diff
@@ -223,38 +223,54 @@ func (d *Driver) handleFingerprint(ctx context.Context, ch chan<- *drivers.Finge
 }
 
 func (d *Driver) buildFingerprint() *drivers.Fingerprint {
-	var health drivers.HealthState
-	var desc string
 	attrs := map[string]*pstructs.Attribute{}
 
-	// be negative and guess that we will not be able to get a podman connection
-	health = drivers.HealthStateUndetected
-	desc = "disabled"
-
-	// try to connect and get version info
-	info, err := d.podman.SystemInfo(d.ctx)
-	if err != nil {
-		d.logger.Error("Could not get podman info", "error", err)
+	// Ping podman api
+	version, err := d.podman.Ping(d.ctx)
+	if err != nil || version == "" {
+		// not reachable?
+		// deactivate driver, forget podman details
+		d.systemInfo = api.Info{}
+		d.logger.Error("Could not get podman version", "error", err)
+		return &drivers.Fingerprint{
+			Attributes:        attrs,
+			Health:            drivers.HealthStateUndetected,
+			HealthDescription: "disabled",
+		}
 	} else {
```

Comment:
Minot nit-picking: I think this `else` is unnecessary since the `if` body will `return`? This would make the rest of the code less indented 🙂

Reply:
Reasonable. Refactored code multiple times and did not see this simplification, thx.